// See README.txt for information and build instructions.

package machines;

message MachineConfig {
    required string name = 1;
    required string volatility_profile = 2;
}

message Machine {
  required int32 ALLOCATED = 1;
  required MachineConfig config = 2;
  required int32 type = 3;
}

message MachineList {
    repeated Machine machine = 1;
}
